pkg/api/defaults.go

```diff
@@ -99,5 +99,6 @@ func DefaultOperatorFlags() OperatorFlags {
 		"rh.srep.muo.managed":                      flagTrue,
 		"aro.guardrails.enabled":                   flagFalse,
 		"aro.guardrails.deploy.managed":            flagFalse,
+		"aro.cloudproviderconfig.enabled":          flagTrue,
```

Comment:
@bennerv based on our discussion, does this need to be `flagFalse` to merge?

Reply:
My understanding is that this should only be for new cluster installs by default because its a new flag (?)

Reply:
True should be fine; it'll be set on new clusters only, and when we PUCM existing clusters we'll patch in their flag to true since it won't be set.

Reply:
on top of that, after PUCM, we should not see a change for existing cluster unless the services are recreated. That being said, it might be worth sending notification about the possible change of outbound ip if services are recreated